### ISSUE TITLE
Warning: Single fetch-enabled apps should not be making ?_data requests, this is likely to break in the future

### ISSUE THREAD

**Issue by kettanaito** [kettanaito/remix-og-image#31]
The latest Remix prints the following warning on `pnpm build` with this plugin:

```
Error: Warning: Single fetch-enabled apps should not be making ?_data requests, this is likely to break in the future
    at requestHandler (/remix-og-image-demo/node_modules/.pnpm/@remix-run+server-runtime@2.14.0_typescript@5.6.3/node_modules/@remix-run/server-runtime/dist/server.js:115:21)
    at nodeHandler (/remix-og-image-demo/node_modules/.pnpm/@remix-run+dev@2.14.0_@remix-run+react@2.14.0_@remix-run+serve@2.14.0_typescript@5.6.3_vite@5.4.11/node_modules/@remix-run/dev/dist/vite/plugin.js:868:33)
    at /remix-og-image-demo/node_modules/.pnpm/@remix-run+dev@2.14.0_@remix-run+react@2.14.0_@remix-run+serve@2.14.0_typescript@5.6.3_vite@5.4.11/node_modules/@remix-run/dev/dist/vite/plugin.js:871:15
```

**Comment by kettanaito** [comment]
Despite saying "Warning" this is actually an error in Remix build, and the plugin short-circuits the nearest image in progress when this error is encountered.

**Comment by kettanaito** [comment]
## Released: v0.4.4 🎉

This has been released in v0.4.4!

- 📄 [**Release notes**](https://github.com/kettanaito/remix-og-image/releases/tag/v0.4.4)
- 📦 [npm package](https://www.npmjs.com/package/remix-og-image/v/0.4.4)

Make sure to always update to the latest version (`npm i remix-og-image@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.
